base_local_planner/src/speed_limiters/obstacle_speed_limiter.cpp

```diff
@@ -239,12 +239,16 @@ ObstacleSpeedLimiter::LinearSpeedLimiterResult ObstacleSpeedLimiter::calculateAl
   }
   else
   {
-    result.speed = threeLevelInterpolation(distance_to_obstruction,
-                                    params_.min_range, params_.nominal_range_min,
-                                    params_.nominal_range_max, params_.max_range,
-                                    std::min(params_.min_linear_velocity, max_linear_velocity_),
-                                    std::min(params_.nominal_linear_velocity, max_linear_velocity_),
-                                    max_linear_velocity_);
+    speed = pow(std::fabs(distance_to_obstruction), 1.0 / params_.extended_obstacle_curve);
```

Comment:
Check for divide by 0.  Either don't let the parameter be 0 in the cfg, or check before it is used.

Reply:
Updating in config